### PR TITLE
ClusterClass and managed topologies blog

### PR DESCRIPTION
**Pull request by fabriziopandini**
This PR adds a new blog article describing ClusterClass and managed topologies, a new feature being added to Cluster API hat will greatly simplify how you can provision, upgrade, and operate multiple Kubernetes clusters in a declarative way.

### CODE REVIEW

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (RinkiyaKeDad) [inline content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:3]: Could you please convert this to title case for consistency, that is, `Introducing ClusterClass and Managed Topologies in Cluster API` 😄

**Review by RinkiyaKeDad — COMMENTED** [review]
One minor nit, looks good otherwise :)

**Review by sftim — COMMENTED** [review]
Thanks @fabriziopandini 

I spotted one grammar nit and made a few wording suggestions. I also proposed adding comments to the example that uses `.spec.topology` in a `cluster.x-k8s.io` Cluster.

Also valuable, if you've the time for it: use SVG rather than PNG for images. SVG images are easier to localize and also look sharper on a high-DPI screen.

We can merge this as-is with tech LGTM from SIG Cluster Lifecycle (once we sort out a publication date). If you have the time to make some extra tweaks, that'd help even more.

/hold
publication date not yet set

**Comment by fabriziopandini** [comment]
@RinkiyaKeDad @sftim Thanks for the quick feedback, really, really appreciated!
All comments should be addressed

@neolit123 @vincepri

**Comment by fabriziopandini** [comment]
@CecileRobertMichon PTAL if you have some time

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sbueringer) [inline content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:18]:
```suggestion
With extensibility addressed, the focus shifted to features, like automatic control plane and etcd management, health-based machine remediation, machine rollout strategies and more.
```
nit (not 100% sure)

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sbueringer) [inline content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:38]: @fabriziopandini Can we / Do we want to use our new v1beta1 types here? :)
(I guess depends on timing of v1beta1 / this blog post)

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sbueringer) [inline content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:87]:
```suggestion
The possibilities are endless; you can get a default ClusterClass from the community, “off-the shelf” classes from your vendor of choice, “certified” classes from the platform admin in your company, or even create custom ones for advanced scenarios.
```
nit

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sbueringer) [reply content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:91]: I think I would choose:
```
Managed topologies let you put the power that ClusterClass provides into action.
```

**Review by sbueringer — COMMENTED** [review]
Just a few nits

**Comment by sftim** [comment]
BTW if you future-date an article, and we merge it, automation publishes it on the scheduled date. This is handy for predictable releases etc.

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sftim) [reply content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:91]: :+1: that wording looks good too.

File: content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md
Comment (sftim) [inline content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:16]: nit: CNCF policy for listing 3rd party products and / or vendors is to use either alphabetical order, or a random shuffle.
How about:
```suggestion
The [Cluster API project](https://github.com/kubernetes-sigs/cluster-api/) started three years ago, and the first releases focused on extensibility and implementing a declarative API that allows a seamless experience across infrastructure providers. This was a success with many cloud providers: AWS, Azure, Digital Ocean, GCP, Metal3, vSphere…  and still counting.
```

Comment (sbueringer) [reply content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md:16]: @sftim Out of curiosity: how do you now something uses "random shuffle"? :)

**Comment by fabriziopandini** [comment]
@sftim @sbueringer thanks for feedback
I have also set publication date to 2021-10-04 (one week before KubeCon, but after CAPI v1beta1 release/with some room for finalising this PR)

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (sbueringer) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:16]:
```suggestion
The [Cluster API project](https://github.com/kubernetes-sigs/cluster-api/) started three years ago, and the first releases focused on extensibility and implementing a declarative API that allows a seamless experience across infrastructure providers. This was a success with many cloud providers: AWS, Azure, Digital Ocean, GCP, Metal3, vSphere and still counting.
```
nit

**Review by sbueringer — COMMENTED** [review]
one last nit, otherwise lgtm

Thank you very much for writing this blog!

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (ykakarap) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:20]:
```suggestion
Fast forwarding to 2021, with lots of companies using Cluster API to manage fleets of Kubernetes clusters running workloads in production, the community focused its effort on stabilization of code, APIs, documentation and on extensive test signals which inform Kubernetes releases.
```

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (ykakarap) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:24]:
```suggestion
Enter ClusterClass and Manage Topologies, tada!
```

Comment (ykakarap) [reply content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:24]:
```suggestion
Enter ClusterClass and managed topologies, tada!
```

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (ykakarap) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:30]:
```suggestion
The idea behind ClusterClass is simple: define the shape of your cluster once, and reuse it many times, abstracting the complexities and the internals of a Kubernetes cluster away.
```

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (ykakarap) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:91]:
```suggestion
Managed topologies let you put the power of ClusterClass into action.
```

**Review by ykakarap — COMMENTED** [review]
This looks great @fabriziopandini! 
Thank you 🙂

Some nits:

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (elmiko) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:87]: seems like a couple dictionaries suggest that "off-the-shelf" should have hyphens between all words
```suggestion
The possibilities are endless; you can get a default ClusterClass from the community, “off-the-shelf” classes from your vendor of choice, “certified” classes from the platform admin in your company, or even create custom ones for advanced scenarios.
```

File: content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md
Comment (elmiko) [inline content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md:139]: i would make this plural
```suggestion
Stay tuned for what comes next, and if you have any questions, comments or suggestions:
```

**Review by elmiko — COMMENTED** [review]
content looks great @fabriziopandini !

just a couple minor grammar nits

**Review by elmiko — APPROVED** [review]
thanks Fabrizio
/lgmt

**Pull request merged**
